README.md

```diff
@@ -1,2 +1,306 @@
+
+1. Create a Trust Policy document to allow the role being used for EC2 management & assume kube2iam role:
+
+    ```
+    cat > velero-trust-policy.json <<EOF
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "Service": "ec2.amazonaws.com"
+                },
+                "Action": "sts:AssumeRole"
+            },
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::<AWS_ACCOUNT_ID>:role/<ROLE_CREATED_WHEN_INITIALIZING_KUBE2IAM>"
+                },
+                "Action": "sts:AssumeRole"
+            }
+        ]
+    }
+    EOF
+    ```
+
+2. Create the IAM role:
+
+    ```bash
+    aws iam create-role --role-name velero --assume-role-policy-document file://./velero-trust-policy.json
+    ```
+
+3. Attach policies to give `velero` the necessary permissions:
+
+    ```
+    BUCKET=<YOUR_BUCKET>
+    cat > velero-policy.json <<EOF
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "ec2:DescribeVolumes",
+                    "ec2:DescribeSnapshots",
+                    "ec2:CreateTags",
+                    "ec2:CreateVolume",
+                    "ec2:CreateSnapshot",
+                    "ec2:DeleteSnapshot"
+                ],
+                "Resource": "*"
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:GetObject",
+                    "s3:DeleteObject",
+                    "s3:PutObject",
+                    "s3:AbortMultipartUpload",
+                    "s3:ListMultipartUploadParts"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::${BUCKET}/*"
+                ]
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:ListBucket"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::${BUCKET}"
+                ]
+            }
+        ]
+    }
+    EOF
+    ```
+    ```bash
+    aws iam put-role-policy \
+      --role-name velero \
+      --policy-name velero-policy \
+      --policy-document file://./velero-policy.json
+    ```
+
+4. Use the `--pod-annotations` argument on `velero install` to add the following annotation:
+
+```bash
+velero install \
+    --pod-annotations iam.amazonaws.com/role=arn:aws:iam::<AWS_ACCOUNT_ID>:role/<VELERO_ROLE_NAME> \
+    --provider aws \
+    --bucket $BUCKET \
+    --backup-location-config region=$REGION \
+    --snapshot-location-config region=$REGION \
+    --no-secret
+```
+
+[0]: namespace.md
+[5]: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html
+[6]: api-types/volumesnapshotlocation.md#aws
```

Comment:
link [6] and [21] bring up interesting points - we currently have the provider-specific config options for BSL/VSL in the core Velero docs. I think we'll want to move those out, into the plugin repo.